docs/pages/authentication.md

```diff
@@ -40,11 +48,18 @@ your [ArcGIS Developer Dashboard](https://developers.arcgis.com/api-keys/).
 
 ## OAuth Sign-in
 
-Using a `ClientId` and `ClientSecret`, you can set up your application to use
-[OAuth Authentication](https://developers.arcgis.com/documentation/mapping-apis-and-services/security/oauth-2.0/).
-This secure system will redirect to an Esri login screen, and then back to your application, with a short-lived token
-that can be used in place of the API Token. This is useful for creating an application for a company or government
-agency that already has ArcGIS accounts.
+OAuth is a more secure way to authenticate your users with ArcGIS. It requires that your users have an ArcGIS account.
+You generate a new OAuth `Application` in the [ArcGIS Developer Dashboard](https://developers.arcgis.com/applications/),
+and then store the `ClientId` in your application configuration as `ArcGISAppId`.
```

Comment:
Can we add a bit here about setting the Portal URL as well to support custom enterprise portal logins?